### PR TITLE
[4.0] mod syndicate icon

### PR DESCRIPTION
**Pull request by brianteeman**
replace the really poor quality orange png with an icon. Not just better quality but scalable and stylable

### before
image

### after
image

### CODE REVIEW

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on cadfd48e069597f09a9a2255791da4908644a67c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33678">issues.joomla.org/tracker/joomla-cms/33678</a>.</sub>

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on cadfd48e069597f09a9a2255791da4908644a67c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33678">issues.joomla.org/tracker/joomla-cms/33678</a>.</sub>

**Comment by alikon** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33678">issues.joomla.org/tracker/joomla-cms/33678</a>.</sub>

File: modules/mod_syndicate/tmpl/default.php [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 ?>
 <a href="<?php echo $link; ?>" class="mod-syndicate syndicate-module">
-	<?php echo HTMLHelper::_('image', 'system/livemarks.png', '', null, true); ?>
+	<span class="icon-feed" aria-hidden="true"></span>
```
Comment (Quy) [inline modules/mod_syndicate/tmpl/default.php:16]: Please remove obsolete use statement.

Comment (brianteeman) [reply modules/mod_syndicate/tmpl/default.php:16]: done

**Comment by richard67** [comment]
I've restored the previous test results in the issue tracker so they are properly counted again. The commit which invalidated the tests was just the removal of a "use" statement of which I've just verified that it's really not needed anymore.

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33678">issues.joomla.org/tracker/joomla-cms/33678</a>.</sub>

**Comment by richard67** [comment]
Thanks!

**Comment by brianteeman** [comment]
thanks

**Pull request merged**